ckanext/versions/logic/dataset_version_action.py

```diff
@@ -116,8 +117,18 @@ def dataset_version_restore(context, data_dict):
     version = version_show(context, data_dict)
     if not version:
         raise toolkit.ObjectNotFound("Version not found")
-    v_name = "restored_{}".format(version['name'])
-    v_notes = "Restored from version: {}".format(version['name'])
+
+    v_name_prefix = _("restored")
+    v_name = "{}_{}".format(v_name_prefix, version['name'])
+    v_notes = _("Restored from version: {}").format(version['name'])
+
+    releases = dataset_version_list(context, {'dataset_id': dataset_name_or_id})
+    release_names = [v['name'] for v in releases]
```

Comment:
that should be a set `{v['name'] for v in releases}`

Reply:
Presumably just to reflect the fact that the releases list is actually unordered?